Added Screenshot to readme (#188)
PaulJonasJost/PEtab_GUI: README.md

Optionally, you can provide the path to an existing PEtab YAML file
as an argument.

> When launching with a problem, PEtabGUI will look like this (up to the position of
> widgets):
> ![PEtabGUI Screenshot](https://raw.githubusercontent.com/PaulJonasJost/PEtab_GUI/screenshot/docs/source/_static/Application_Screenshot.png)

Reviewer: ```suggestion
After loading a PEtab problem, PEtabGUI will look something like this:
![PEtabGUI Screenshot](https://raw.githubusercontent.com/PaulJonasJost/PEtab_GUI/screenshot/docs/source/_static/Application_Screenshot.png)
```

(up to position, and content, and ...  😅 )
